src/Core/InstanceConfigTrait.php

```diff
@@ -296,7 +297,7 @@ protected function _configDelete(string $key): void
 
         foreach ($stack as $i => $k) {
             if (!is_array($update)) {
-                throw new Exception(sprintf('Cannot unset %s value', $key));
+                throw new RuntimeException(sprintf('Cannot unset %s value', $key));
```

Comment:
This is an unnecessary change. The exception here is `\Cake\Core\Exception\Exception` which extends `\RuntimeException`, it's not PHP's `\Exception`.

Reply:
Ah, you are right about this.